pavelib/prereqs.py

```diff
@@ -195,22 +195,12 @@ def uninstall_python_packages():
     # to really really get rid of it.
     for _ in range(3):
         uninstalled = False
-        frozen = sh("pip freeze", capture=True).splitlines()
-
-        # Uninstall South
-        if any(line.startswith("South") for line in frozen):
-            sh("pip uninstall --disable-pip-version-check -y South")
-            uninstalled = True
-
-        # Uninstall edx-val
-        if any("edxval" in line for line in frozen):
-            sh("pip uninstall --disable-pip-version-check -y edxval")
-            uninstalled = True
-
-        # Uninstall django-storages
-        if any("django-storages==" in line for line in frozen):
-            sh("pip uninstall --disable-pip-version-check -y django-storages")
-            uninstalled = True
+        frozen = sh("pip freeze", capture=True)
+
+        for package in ("South", "edxval", "django-storages", "django-oauth2-provider"):
+            if "{}==".format(package) in frozen:
```

Comment:
Based on your statements, it sounds like this code is no longer needed. If I updated it to simply remove `django-oauth2-provider`, would that be okay?